javascript-data-model/order-history.js

```diff
@@ -0,0 +1,100 @@
+var item = {
+  title: String,
+  author: String,
+  image: String,
+  price: Number,
+  returnWindow: Date
+};
+
+var order = {
+  items: [item],
+  orderId: String,
+  orderDate: Date,
+  orderTotal: Number,
+  shipTo: String,
+  deliveryDate: Date,
+  deliveryNotes: String
+};
```

Comment:
These can be removed. Data types do not need to be declared in advance in JavaScript

Reply:
Really? Interesting okay thank you for the feedback Cody! I will fix that right now.